gp-includes/things/translation.php

```diff
@@ -163,7 +168,12 @@ public function for_translation( $project, $translation_set, $page, $filters = a
 		$statuses = explode( '_or_', $status );
 		if ( in_array( 'untranslated', $statuses ) ) {
 			if ( $statuses == array( 'untranslated' ) ) {
-				$where[] = 't.translation_0 IS NULL';
+				$empty_plural_translation = '(' . implode( ' OR ', array_map(
+					function( $x ) { return "$x = ''"; },
+					array( 't.translation_0', 't.translation_1', 't.translation_2', 't.translation_3', 't.translation_4', 't.translation_5', )
+				) ) . ')';
+
+				$where[] = "t.translation_0 IS NULL OR (o.plural IS NOT NULL AND $empty_plural_translation)";
```

Comment:
However if it's built in to the upgrade process the CLI isn't needed, unless the routine fails during the upgrade, I which case there should be a way to re-run it from the GUI if they have increased their memory limit.

My concern is that most users aren't familiar with the CLI and many do no have access to it.